pkg/kubelet/metrics/metrics.go

```diff
@@ -573,6 +574,18 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// TopologyManagerAdmissionDuration is a Histogram that tracks the duration (in seconds) to serve a pod admission request.
+	TopologyManagerAdmissionDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           TopologyManagerAdmissionDurationKey,
+			Help:           "Duration in seconds to serve a pod admission request.",
```

Comment:
I expect the admission to take fraction of seconds, more in the realm of tenths of milliseconds. I'm inclined to think we should change the unit from `seconds` to `milliseconds`, WDYT?
Perhaps let's get some examples from actual e2e test runs (CI, local environments) before.

Reply:
Good point, Thanks. I will play around with e2e tests and update.